libs/wingc/src/lifting.rs

```diff
@@ -285,16 +285,6 @@ impl<'a> Visit<'a> for LiftVisitor<'a> {
 			None
 		};
 
-		if self.ctx.current_phase() == Phase::Inflight {
```

Comment:
Note this is dead because it'll reach this code only if the current phase is inflight and the visited class is a preflight class. But it's not possible to define a preflight class within an inflight context. I'm not sure what we were trying to achieve here.
@eladb?

Reply:
Safe to delete. It's a leftover I hadn't had a chance to clean up!